iphone/Classes/AppModule.m

```diff
@@ -188,7 +188,7 @@ - (void)fireEvent:(NSArray *)args
 
     DebugLog(@"[DEBUG] Firing app event: %@", type);
 
-    NSArray *array = [appListeners objectForKey:type];
+    NSArray *array = [[appListeners objectForKey:type] copy];
```

Comment:
I wonder if we should rather synchronize the `appListeners` setters, so nothing can be mutated during its write-process.

Reply:
Ok then, taking your word!